### DISCUSSION TITLE
Asking About your answer for my application to your program GitHub expert !!

### DISCUSSION THREAD

**Discussion by sher0u** [community/community#50205] [GitHub Education] [answered]
Hello ..
I applied for Your program campus GitHub expert , and after the application I read that the answer will be in the 15 of the current month which is March  , and today we are 16 and I didn't receive any answer. I hope you will answer me quickly .

**Comment by ShubhadeepKarmakar** [comment]
Same here.

**Comment by lasr21** [comment, marked as answer]
Hi there! We wanted to let you know that an email has been sent out to everyone. We also wanted to remind you that you can check the status of your application on https://education.github.com/campus_experts/status. Wishing you all the best!

**Comment by sher0u** [comment]
спасибо большое 😄

чт, 16 мар. 2023 г., 23:58 lasr21 ***@***.***>:
